src/lib/array.js

```diff
@@ -48,6 +51,128 @@ exports.ensureArray = function(out, n) {
     return out;
 };
 
+function detectType(a) {
+    return typeof a === 'undefined' ? undefined : a;
+}
+
+var typedArrays = {
+    i1: detectType(Int8Array),
+    u1: detectType(Uint8Array),
+    c1: detectType(Uint8ClampedArray), // not supported in numpy?
+    i2: detectType(Int16Array),
+    u2: detectType(Uint16Array),
+    i4: detectType(Int32Array),
+    u4: detectType(Uint32Array),
+    f4: detectType(Float32Array),
+    f8: detectType(Float64Array),
+
+    // TODO: potentially add Big Int
+    // I8: detectType(BigInt64Array), // not supported in numpy?
+    // b8: detectType(BigUint64Array),
```

Comment:
numpy does support [int64](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.int64) (`i8`) and [unit64](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.uint64) (`u8`)

Reply:
Comment corrected in [1230ba8](https://github.com/plotly/plotly.js/pull/5230/commits/1230ba8215294f730d191866be363bfdd5cfa8e6).